tools/experiment.py

```diff
@@ -919,9 +918,8 @@ def _prepare_run_command(self, command: List[str]) -> bool:
         return True
 
     def _run_otx_command(self, command: List[str]):
-        sys.argv = copy(command)
         try:
-            otx_cli()
+            subprocess.run(command, check=True)
```

Comment:
Is it for stability?

Reply:
Yes, it is. Multiple otc_cli() call resulted in CPU OOM at times even though I'd tried `gc.collect()`.
There might be some memory leak.